config/application.rb

```diff
@@ -27,6 +27,11 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    config.log_level = :debug
+    STDOUT.sync = true # turn off log buffering
```

Comment:
reposting question from yesterday: at first glance this seems to make all logging requests synchronous writes with the logger. This means IO interrupts during request processing and would be potentially very bad for performance.  Do these writes get buffered somewhere else or is there some other mitigating factor?